### PR TITLE
376: A11y Testing ADR

### PR DESCRIPTION
**Pull request by halprin**
## What changed

An ADR for a11y testing.

## Issue

#376.

### CODE REVIEW

File: docs/adr/015-a11y-testing.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Status
+
+Accepted.
+
+## Context
+
+A11y is important.  Not only that, it is mandated by federal law.
+
+[aXe](https://www.deque.com/axe/) is a major player in the a11y testing and training.  They make an open-source tool called axe-core that we'll
```
Comment (tdonaworth) [inline docs/adr/015-a11y-testing.md:20]: Do we have any specific reason for choosing `axe` over something else? I feel like thats one of the main purposes of these records as well is to enforce why the decision was made over something else.

Comment (halprin) [reply docs/adr/015-a11y-testing.md:20]: That's a good point.  I _heard_ that aXe is better than pa11y from Brice.  Let me dig that up, do some more research, and expand on the ADR.

File: docs/adr/015-a11y-testing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+A11y is important.  Not only that, it is mandated by federal law.
+
+[aXe](https://www.deque.com/axe/) is a major player in the a11y testing and training.  They make an open-source tool
+called axe-core that we'll be utilizing through Cypress.  That makes it easy to a11y test because Cypress is...
+
+- Adept at interacting and navigating websites to get them into all the different states that need to be a11y tested.
+- Already used for our [end-to-end testing](./014-use-cypress-for-testing.md).
+
+We' are using aXe because it...
```
Comment (tdonaworth) [inline docs/adr/015-a11y-testing.md:26]: "We're"?

**Review by tdonaworth — APPROVED** [review]
fix typo and I'm good

**Review by johndeange — APPROVED** [review]

**Pull request merged**